src/containers/Footer.js

```diff
@@ -6,6 +6,23 @@ import { AppFooter } from '@coreui/react';
 export function Footer(props) {
 
 	let FooterService = props.app.locateService("FooterService");
+	let siteFooterImage = props.app.Config.get("footer_image");
+
+	let footerImage = props.footer_image.src;
+	let footerImageAlt = props.footer_image.alt;
+	let footerImageHref = props.footer_image.href;
+
+	if (siteFooterImage !== undefined) {
```

Comment:
This is unnecessary code - props.footer_image is props.app.Config.get("footer_image") so this whole `if` section is useless.